tests/core/server/test_dos.py

```diff
@@ -49,14 +49,14 @@ async def test_large_message_disconnect_and_ban(self, setup_two_nodes_fixture, s
 
         ssl_context = server_2.ssl_client_context
         ws = await session.ws_connect(
-            url, autoclose=True, autoping=True, heartbeat=60, ssl=ssl_context, max_msg_size=100 * 1024 * 1024
+            url, autoclose=True, autoping=True, ssl=ssl_context, max_msg_size=100 * 1024 * 1024
```

Comment:
So we're disabling the heartbeat for these tests?  Seems appropriate to keep it on just for consistency in the tests?

Reply:
> So we're disabling the heartbeat for these tests? Seems appropriate to keep it on just for consistency in the tests?
yeah, why are we doing this?

Reply:
My rationale was that the tests don't actually test the heartbeat mechanism, nor our handling of it, so why set it when I think all it can do is cause flakes.

Reply:
I'll accept that.  It's an inaccuracy but it has a purpose and it's a library feature.  If I get a thumbs up from all I'll close this conversation.  Otherwise, say something more.

Reply:
Well, I'm not married to the idea either -